docs/manual/java/guide/cluster/ReadSideJDBC.md

```diff
@@ -0,0 +1,85 @@
+# JDBC Read-Side support
```

Comment:
This page is mostly a copy of the previous `ReadSideRDBMS.md` with a few minor changes.